### PR TITLE
Feature/create page my profile education

### PR DESCRIPTION
**Pull request by LittleSweetGun**
Add table users_education

CREATE TABLE goodhive.companies (
id SERIAL PRIMARY KEY,
headline VARCHAR(255) NOT NULL,
designation VARCHAR(100) NOT NULL,
address VARCHAR(100) NOT NULL,
country VARCHAR(100) NOT NULL,
city VARCHAR(100) NOT NULL,
phone_country_code VARCHAR(10) NOT NULL,
phone_number VARCHAR(20) NOT NULL,
email VARCHAR(255) NOT NULL UNIQUE,
telegram VARCHAR(255),
details TEXT NOT NULL,
image_url VARCHAR(255)
);

TODO:
- choose which `input` have to be `select input`

### CODE REVIEW

**Comment by LittleSweetGun** [comment]
#87

**Comment by LittleSweetGun** [comment]
See which inputs have to be select-inputs

File: README.md [hunk 1]
```diff
@@ -34,3 +34,75 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
...
+id SERIAL PRIMARY KEY,
+school VARCHAR(100) NOT NULL,
+degree VARCHAR(100) NOT NULL,
+filed VARCHAR(100) NOT NULL,
+location VARCHAR(100) NOT NULL,
+start_date VARCHAR(100) NOT NULL,
+end_date VARCHAR(100) NOT NULL,
+description VARCHAR(255),
+distinction VARCHAR(100)
+);
```
Comment (n1c01a5) [inline README.md:108]: Move this to a `init.sql` file.

File: app/talents/my-profile/education/page.tsx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+    const startYear = Number(selectedStartYear?.value);
+    const startMonth = Number(selectedStartMonth?.value);
+    const startDate = new Date(startYear, startMonth);
+
+    const endYear = Number(selectedEndYear?.value);
+    const endMonth = Number(selectedEndMonth?.value);
+    const endDate = new Date(endYear, endMonth);
+
+    const dataForm = {
```
Comment (n1c01a5) [inline app/talents/my-profile/education/page.tsx:41]: It's a generic name. `educationForm` is more explicit.

It's more useful if we have multiple form. It's not the case but it's better to anticipate this.

**Review by n1c01a5 — APPROVED** [review]

**Pull request merged**